docs: Update get-started-next-steps.md (#14602)
unoplatform/uno: doc/articles/get-started-next-steps.md

## Uno Platform provided Worshops / Step-by-step Tutorials (Free)

- [Simple Calculator Workshop](external/workshops/simple-calc/README.md). A level 100 workshop you can complete in 1 hour. It will guide you through the process of setting up your developer environment, learning essential tools, libraries, and patterns to help you develop your apps with Uno Platform.
- [Tube Player Workshop](external/workshops/simple-calc/README.md). A level 200 workshop you can complete in 2 hours. It will guide through the creation of an Uno Platform application that enables the user to search for and stream, YouTube videos. You will learn how to easily import Figma designs into your application.

## Community Provided Learning Resources (Free and Paid)

#### Syncfusion - Uno Platform Succinctly eBook (Free)

Syncfusion published [Uno Platform Succinctly]( https://www.syncfusion.com/succinctly-free-ebooks/uno-platform-succinctly). A free eBook that will take you from setting up your first Uno Platform project with Syncfusion UI Controls in Visual Studio to a fully functioning single-page application.

#### Pluralsight - Uno Platform Fundamentals (Paid)

> If you already have a subscription to Pluralsight, you can find our [Uno Platform Fundamentals course](https://www.pluralsight.com/courses/uno-platform-fundamentals). By the end of the course, you will have created your first Uno Platform solution in Visual Studio and you’ll have the skills and knowledge needed to build multi-platform apps with Uno Platform. 
> 
> The course covers how to build a user interface in XAML, how to apply the MVVM pattern, how to adapt your user interface to different screen sizes, how to write platform-specific code and how to connect your application to a data source. _If you’re not Pluralsight subscriber, you can still use the course it with a 30-day trial._

Reviewer: ```suggestion
If you already have a subscription to Pluralsight, you can find our [Uno Platform Fundamentals course](https://www.pluralsight.com/courses/uno-platform-fundamentals). By the end of the course, you will have created your first Uno Platform solution in Visual Studio, and you’ll have the skills and knowledge needed to build multi-platform apps with Uno Platform. 

The course covers how to build a user interface in XAML, how to apply the MVVM pattern, how to adapt your user interface to different screen sizes, how to write platform-specific code and how to connect your application to a data source. _If you’re not a Pluralsight subscriber, you can still use the course with a 30-day trial._
```

   8. [Uno Platform WASM Invoke JavaScript]( https://www.andrewhoefling.com/Blog/Post/uno-platform-wasm-invoke-javascript)

#### Building a Todo App with Uno Platform (Free)

> [Steven Giesel](https://github.com/linkdotnet). explored the in’s and out of Uno Platform and in turn documented his experience with a great blog series that dives further into What is Uno Platform and how to build a multi-platform Todo App.

Reviewer: ```suggestion
[Steven Giesel](https://github.com/linkdotnet) explored the ins and outs of Uno Platform and in turn, documented his experience with a great blog series that dives further into What is Uno Platform and how to build a multi-platform Todo App.
```

#### XAML Fundamentals for Web and Mobile (Free)

Any UWP/WinUI/XAML material is a great starting point as all we do is take that over the Uno Platform bridge. To that point, one Uno Platform contributor and Microsoft MVP, Nick Randolph has been revitalizing older XAML resources on his [.NET Travels blog.](https://nicksnettravels.builttoroam.com/xaml-basics/) Uno Platform will be continuing the series on our blog.

## Microsoft docs and blogs on Uno Platform (Free) 

- WinUI Documentation - Basic (2 posts) [Build a Hello World app using C# and WinUI 3 / Windows App SDK](https://learn.microsoft.com/en-us/windows/apps/how-tos/hello-world-winui3) leads you through setup of a very basic WinUI Application. The second blog post [How to target multiple platforms with your WinUI 3 app](https://learn.microsoft.com/en-us/windows/apps/how-tos/hello-world-winui3) walks you through taking that same Hello World application cross-platform using Uno PLatform.

> - WinUI Documentation - Advanced (2 posts) [Tutorial: Create a simple photo viewer with WinUI 3](https://learn.microsoft.com/en-us/windows/apps/get-started/simple-photo-viewer-winui3?tabs=cs) leads you through creation of a  WinUI-based Photo Viewing Application. The second blog post [Tutorial: Build a simple photo viewer that targets multiple platforms](https://learn.microsoft.com/en-us/windows/apps/get-started/uno-simple-photo-viewer) walks you through taking the Photi Viewing application cross-platform using Uno PLatform.

Reviewer: ```suggestion
- WinUI Documentation - Advanced (2 posts) [Tutorial: Create a simple photo viewer with WinUI 3](https://learn.microsoft.com/en-us/windows/apps/get-started/simple-photo-viewer-winui3?tabs=cs) leads you through creation of a  WinUI-based Photo Viewing Application. The second blog post [Tutorial: Build a simple photo viewer that targets multiple platforms](https://learn.microsoft.com/en-us/windows/apps/get-started/uno-simple-photo-viewer) walks you through taking the Photo Viewing application cross-platform using Uno PLatform.
```
